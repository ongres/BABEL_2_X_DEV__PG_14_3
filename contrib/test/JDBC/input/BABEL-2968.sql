use master;
go

create table t2968 (name int);
insert into t2968 values (1);
go

CREATE VIEW  v2968 AS
---these are “smart quotes”
SELECT name objectName FROM t2968
go

select * from v2968;
go

drop view v2968;
go
drop table t2968;
go
