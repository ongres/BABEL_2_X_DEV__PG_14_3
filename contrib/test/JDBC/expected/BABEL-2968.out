use master;
go

create table t2968 (name int);
insert into t2968 values (1);
go
~~ROW COUNT: 1~~


CREATE VIEW  v2968 AS
---these are “smart quotes”
SELECT name objectName FROM t2968
go

select * from v2968;
go
~~START~~
int
1
~~END~~


drop view v2968;
go
drop table t2968;
go
